bootkube/resources/charts/kube-apiserver/values.yaml

```diff
@@ -0,0 +1,19 @@
+apiserver:
+  apiserverKey:
+  apiserverCert:
+  serviceAccountPub:
+  caCert:
+  etcdClientCaCert:
+  etcdClientCert:
+  etcdClientKey:
+  aggregationCaCert:
+  aggregationClientCert:
+  aggregationClientKey:
+  image: k8s.gcr.io/hyperkube:v1.17.3
+  cloudProvider:
+  etcdServers:
+  aggregationFlags:
+  serviceCIDR: 10.0.0.0/24
+  trustedCertsDir: /usr/share/ca-certificates
+podCheckpointer:
```

Comment:
Why is this part of apiserver chart?

Reply:
Because pod checkpointer is currently only used by kube-apiserver. Perhaps it would make sense to separate them, but I think this can be done later.